uportal-portlets-overlay/NotificationPortlet/src/main/resources/configuration.properties

```diff
@@ -18,12 +18,40 @@
 #
 
 
-## TIP:  You can place configuration elements meaningful to your
-## NotificationService implementations in this file and use Maven 
-## filtering to set them.
-
 UsernameFinder.unauthenticatedUsername=guest
 
+finance.endpoint.url=${configuration.finance.endpoint.url}
+finance.endpoint.username=${configuration.finance.endpoint.username}
+finance.endpoint.password=${configuration.finance.endpoint.password}
+
+
+remedyService.username=${configuration.remedyService.username}
+remedyService.password=${configuration.remedyService.password}
+remedyService.locale=${configuration.remedyService.locale}
+remedyService.server=${configuration.remedyService.server}
+remedyService.port=${configuration.remedyService.port}
+remedyService.requestType=${configuration.remedyService.requestType}
+remedyService.baseUrl=${configuration.remedyService.baseUrl}
+
+
+remedyGroupService.username=${configuration.remedyGroupService.username}
+remedyGroupService.password=${configuration.remedyGroupService.password}
+remedyGroupService.locale=${configuration.remedyGroupService.locale}
+remedyGroupService.server=${configuration.remedyGroupService.server}
+remedyGroupService.port=${configuration.remedyGroupService.port}
+remedyGroupService.requestType=${configuration.remedyGroupService.requestType}
+remedyGroupService.baseUrl=${configuration.remedyGroupService.baseUrl}
```

Comment:
Guys these properties are a part of some school's implementation of Notification.  They've crept into Apereo versions of this files accidentally.  Please just remove them.  ("$ grep -R "emedy" *" in an up-to-date clone of Notification turns up hits only in this file.)